### PR TITLE
Springster styling phase 2

### PR DESCRIPTION
**Pull request by projectmushroom**

### CODE REVIEW

File: gem/styles/gem-springster/modules/_m-banner.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 
 .banner__item-link {
   display: block;
+  img {
```
Comment (Mitso) [inline gem/styles/gem-springster/modules/_m-banner.scss]: We should add a class for banner images here to reuse styles as much as possible. 
My bad

File: gem/styles/gem-springster/modules/_m-tag-cloud.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    position: absolute;
+    left: 50%;
+    margin-left: -50px;
+  }
+
+  &__heading {
+    @include zig-zag-underline;
+    text-align: center;
+
+    h4 {
```
Comment (Mitso) [inline gem/styles/gem-springster/modules/_m-tag-cloud.scss]: I saw the tag cloud had different variations on articles as well. On clean up we must be able to pick and choose pattern elements without loosing styles.

**Comment by Mitso** [comment]
@projectmushroom just a few comments to keep in mind when we do a cleanup I think. Otherwise it's 💯 for land this.

File: gem/styles/gem-springster/modules/_m-tag-cloud.scss (hunk 2 above)
Comment (projectmushroom) [inline gem/styles/gem-springster/modules/_m-tag-cloud.scss]: cool we can do these two in clean up, the buttons styles also needs some work

**Pull request merged**